modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -205,6 +220,7 @@ private void _writeBody(JspWriter jspWriter) throws Exception {
 		if ((href != null) && !disabled) {
 			LinkTag linkTag = new LinkTag();
 
+			linkTag.setLocalized(isLocalized());
```

Comment:
I agree with what you said in https://github.com/liferay-frontend/liferay-portal/pull/2917#issuecomment-1430744483, checking null seems the best option. Something like

```suggestion
			Boolean localized = isLocalized();
			
			if (localized != null) {
			    linkTag.setLocalized(localized);
			}
```

We want to keep default of LinkTag to be `true`, and for HorizontalCard to inherit that default.